projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.ts

```diff
@@ -379,6 +450,21 @@ export class IgxTreeGridComponent extends IgxGridBaseDirective implements GridTy
         super.ngDoCheck();
     }
 
+    /**
+     * @hidden
+     */
+    public ngAfterViewInit() {
+        super.ngAfterViewInit();
+        // TODO: pipesExectured event
+        // run after change detection in super triggers pipes for records structure
+        if (this.rowSelection === 'multipleCascade' && this.selectedRows.length) {
```

Comment:
Checks should be performed against the enum, rather than against the string representation. This way we can change the string without having to find and replace it everywhere!